libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/RouteOptionsNavigation.kt

```diff
@@ -184,18 +187,10 @@ class RouteOptionsNavigation private constructor(
             )
         }
 
-        private fun assembleCoordinates() {
-            _origin?.let { origin ->
-                coordinates.add(origin)
-            }
-
-            for (waypoint in _waypoints) {
-                coordinates.add(waypoint)
-            }
+        private fun checkFields() {
```

Comment:
Apart from `origin` and `destination`, are there any other mandatory fields that we should check?

Reply:
thank for note!